mos-platform/commodore/cbm_k_acptr.s

```diff
@@ -0,0 +1,10 @@
+.text
+
+#include <cbm_kernal.inc>
+
+;
+; unsigned char cbm_k_acptr (void);
+;
+.global cbm_k_acptr
+cbm_k_acptr:

```

Comment:
Optional: if you have time, it would be better to write as many of these functions as possible in C. This would allow thin wrappers like this to be completely inlined at the point of use: a call to cbm_k_acptr would just become jsr ACPTR inside the user code.

In C, it would look like this:
```C
unsigned char __ACPTR(void);
unsigned char cbm_k_acptr(void) {
  return __ACPTR();
}
```

No action required; I can always take care of this later, as time permits.